scripts/documentation/package-schemas/package-metadata.ts

```diff
@@ -125,6 +125,9 @@ export function findPackageMetadataList(
       const folderName = folderPath.substring(packagesDir.length + 1);
 
       const relativeFolderPath = folderPath.replace(absoluteRoot, '');
+      if (!existsSync(join(folderPath, 'package.json'))) {
```

Comment:
What's the effect of returning `null` here? Does this need to be guarded elsewhere?

Reply:
After the map, we do a `.filter(Boolean)` to remove all the `null`s

Reply:
I could swap the order of the filter and the map if you prefer